AUTHORS.md

```diff
@@ -29,3 +29,5 @@ pieces of artwork in this repository.
 - [`music/kleines Filmröllchen - Just FLACing Around.flac`](music/kleines%20Filmröllchen%20-%20Just%20FLACing%20Around.flac) - [@kleinesfilmroellchen](https://github.com/kleinesfilmroellchen)
 - [`music/kleines Filmröllchen - Bitter Rain (Live Performance).mp3`](music/kleines%20Filmröllchen%20-%20Bitter%20Rain%20(Live%20Performance).mp3) - [@kleinesfilmroellchen](https://github.com/kleinesfilmroellchen)
 - [`music/kleines Filmröllchen - Bitter Rain.flac`](music/kleines%20Filmröllchen%20-%20Bitter%20Rain.flac) - [@kleinesfilmroellchen](https://github.com/kleinesfilmroellchen)
+- [`videos/retro_serenity0001-0120.mp4`](videos/retro_serenity0001-0120.mp4) - [@xslendix](https://github.com/xslendix)
```

Comment:
:skeleyak: but,

Maybe come up with a slightly better name than `retro_serenity0001-0120.mp4`; that looks like what the export function of whatever you used wanted to call it.

:^)